src/__tests__/plugin.js

```diff
@@ -51,17 +51,17 @@ describe('gatsby-remark-embedder', () => {
       <iframe src="https://codepen.io/team/codepen/embed/preview/PNaGbb" style="width:100%; height:300px;"></iframe>
       <iframe src="https://codesandbox.io/embed/ynn88nx9x?view=split" style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;" allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking" sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"></iframe>
       <div style="width:100%;height:0;padding-bottom:63%;position:relative;"><iframe src="https://giphy.com/embed/XatG8bioEwwVO" width="100%" height="100%" style="position:absolute" frameborder="0" class="giphy-embed-from-cache" allowfullscreen></iframe></div>
-      <blockquote class="instagram-media-from-cache"><div><a href="https://instagram.com/p/B60jPE6J8U-"><p>example</p></a><p>A post shared by <a href="https://instagram.com/michaeldeboey">Michaël De Boey</a> (@michaeldeboey) on<timedatetime="2020-01-02T14:45:30+00:00">Jan 2, 2020 at 6:45am PST</time></p></div></blockquote>
+      <blockquote class="instagram-media-from-cache"><div><a href="https://instagram.com/p/B60jPE6J8U-"><p>example</p></a><p>A post shared by <a href="https://instagram.com/michaeldeboey">Michaël De Boey</a> (@michaeldeboey) on<timedatetime="2020-01-02t14:45:30+00:00">Jan 2, 2020 at 6:45am PST</timedatetime="2020-01-02t14:45:30+00:00"></p></div></blockquote>
```

Comment:
This `<timedatetime...>` “element” (which was already there), does seem a bit weird btw. Maybe something in the fetching is up?

Reply:
That's coming from our mock. I'm guessing that's a typo, so I fixed it: https://github.com/MichaelDeBoey/gatsby-remark-embedder/commit/6f0f7d1017dad5a1530204d35ece19c10a5288d8

Thanks!